spec/features/search/cd_api/case_reference_spec.rb

```diff
@@ -54,4 +54,39 @@
 
     expect(page).to be_accessible.within '#main-content'
   end
+
+  scenario 'with error from common_platform', :stub_defendants_cda_failed do
+    visit '/'
+
+    choose 'A case by URN'
+    click_button 'Continue'
+    fill_in 'search-term-field', with: 'error'
+    click_button 'Search'
+
+    expect(page).not_to have_css('.govuk-body', text: 'There are no matching results')
+    expect(page).to have_css('.govuk-error-summary')
+    within '.govuk-error-summary' do
+      expect(page).to have_content('Unable to complete the search. Please try again in a moment.')
+    end
+
+    expect(page).to be_accessible.within '#main-content'
+  end
+
+  scenario 'with error from CDA', :stub_defendants_failed_search do
+    visit '/'
+
+    choose 'A case by URN'
+    click_button 'Continue'
+    fill_in 'search-term-field', with: 'error'
+    click_button 'Search'
+
+    expect(page).not_to have_css('.govuk-body', text: 'There are no matching results')
+    expect(page).to have_css('.govuk-error-summary')
+    within '.govuk-error-summary' do
+      expect(page)
+        .to have_content('Unable to complete the search. Please adjust the search terms and try again.')
+    end
```

Comment:
is it worth testing the IT helpdesk description  as well `I18n.t('error.it_helpdesk')` ?